Automattic/wp-calypso
# Remove files and dead code and redirect to actual DIFM

#### Changes proposed in this Pull Request

* Cleanup DIFM related dead code in the marketing section
* Redirect all `/marketing/do-it-for-me` routes to the marketing landing page
* Cleanup the feature flag

#### Testing instructions

* Visit `/marketing/do-it-for-me/landing` , or any child of `/marketing/do-it-for-me`, You should be redirected to https://wordpress.com/do-it-for-me/
* Go to a site with the DIFM in progress sticker and make sure the relevant UI shows
